### ISSUE TITLE
Proposal for new npm maintainer

### ISSUE THREAD

**Issue by mzgoddard** [ColCh/jest-webpack#6]
Hi there, I have a project, https://github.com/mzgoddard/jest-webpack#how-it-works, in active development and I'm curious if you'd be open to me taking over this name on npm since this seems to be an inactive project.

**Comment by ColCh** [comment]
Hi. Thank you for contacting me!
This project seems interesting. At very first stage, current project (ColCh/jest-webpack) meant to be like yours.
So, I agree that we change npm module owner.

Please give me some time to read about name ownership translation. I will notify You when all is ready.

**Comment by mzgoddard** [comment]
@ColCh thank you for responding. I think the easiest way to add an owner/maintainer on `npm` is through the web interface. You can log in and navigate to the package and click edit or the plus button in the collaborators section. https://www.npmjs.com/package/jest-webpack `npm` seems to have many names for the same thing, write access to the package. I think the author field is just an editable value in the `package.json`. There is also the [`npm owner`](https://docs.npmjs.com/cli/owner) command.

**Comment by ColCh** [comment]
It seems to be done:

```
~ $ npm owner ls jest-webpack
mzgoddard <mzgoddard@gmail.com>
colch <colchgh@gmail.com>
```

I've deprecated version 0 of jest-webpack. 

From now You have access to npm module name. 

Good luck! Please close issue if everything is OK

**Comment by mzgoddard** [comment]
@ColCh I released my implementation as 0.3.0 to reduce confusion between it and your prior work. Thanks again!

**Comment by ColCh** [comment]
@mzgoddard glad to hear :) Good luck with your work!
